homework2/reviewapp/src/app/components/features/shows/ShowDetails/ShowDetails.tsx

```diff
@@ -0,0 +1,27 @@
+import { IShow } from "@/app/typings/show";
+import { Card, CardBody, Flex,  Heading,  Image, Text} from "@chakra-ui/react";
+
+export interface IShowItemProps{
+    show: IShow
+
+}
+export const ShowDetails=({show}: IShowItemProps) =>
+{
+
+    return (
+        <Flex flexDirection="column" width="100%" borderRadius="15px"   paddingBottom={5} gap={5} marginBottom={4}   height="fit-content">
```

Comment:
Do we really need flex here?

Reply:
I suppose not because i can just return the Card element